### PR TITLE
disable regresion v1 check

### PR DESCRIPTION
**Pull request by ermirizio**
remove v1 as required check

### CODE REVIEW

**Comment by buger** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/5877/merge
Commit:  
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7226990427)

**Comment by buger** [comment]
**API tests result - postgres15-murmur64 env: success** :white_check_mark: 
Branch used: refs/pull/5877/merge
Commit: 152da0bf9258bb2339929ac3aa364e1ab79e764b 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7227010059)

**Comment by buger** [comment]
**API tests result - mongo44-murmur64 env: success** :white_check_mark: 
Branch used: refs/pull/5877/merge
Commit: 152da0bf9258bb2339929ac3aa364e1ab79e764b 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7227010059)

**Comment by buger** [comment]
**API tests result - mongo44-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/5877/merge
Commit: 152da0bf9258bb2339929ac3aa364e1ab79e764b 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7227010059)

**Comment by buger** [comment]
**API tests result - postgres15-sha256 env: success** :white_check_mark: 
Branch used: refs/pull/5877/merge
Commit: 152da0bf9258bb2339929ac3aa364e1ab79e764b 
Triggered by: pull_request (@ermirizio)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/7227010059)

File: repo-policy/main.tf [hunk 1]
```diff
@@ -49,7 +49,7 @@ module "tyk" {
     { branch    = "master",
       reviewers = "1",
       convos    = "false",
-    required_tests = ["Go 1.21.x Redis 5", "1.21-bullseye", "test (1.21.x, ubuntu-latest, amd64, 15.x)", "api-tests (sha256, mongo44)", "api-tests (sha256, postgres15)", "api-tests (murmur64, mongo44)", "api-tests (murmur64, postgres15)"] },
```
Comment (alephnull) [inline repo-policy/main.tf:52]: I feel that 1.21-bullseye can also be removed as api-tests depend on this job to pass.

**Comment by alephnull** [comment]
I think this is part of SYSE-273 not SYSE-307.

File: repo-policy/main.tf (hunk 1 above)
Comment (ermirizio) [reply repo-policy/main.tf:52]: No @alephnull , 1.21-bullseye is goreleaser job, I don't think we want to remove that

**Comment by ermirizio** [comment]
> I think this is part of SYSE-273 not SYSE-307.

Is actually 288 sub task, I will change it accordinglye

File: repo-policy/main.tf (hunk 1 above)
Comment (alephnull) [reply repo-policy/main.tf:52]: api-tests depends on goreleaser so it seems pointless to keep 1.21-bullseye.

**Pull request closed without merging**